orchagent/portsorch.cpp

```diff
@@ -3593,24 +3799,36 @@ bool PortsOrch::getPortOperStatus(const Port& port, sai_port_oper_status_t& stat
     return true;
 }
 
-bool PortsOrch::removeAclTableGroup(const Port &p)
+bool PortsOrch::getSaiAclBindPointType(Port::Type           type,
+                                       sai_acl_bind_point_type_t &sai_acl_bind_type)
 {
-    sai_acl_bind_point_type_t bind_type;
-    switch (p.m_type)
+    switch(type)
     {
         case Port::PHY:
-            bind_type = SAI_ACL_BIND_POINT_TYPE_PORT;
+            sai_acl_bind_type = SAI_ACL_BIND_POINT_TYPE_PORT;
             break;
         case Port::LAG:
-            bind_type = SAI_ACL_BIND_POINT_TYPE_LAG;
+            sai_acl_bind_type = SAI_ACL_BIND_POINT_TYPE_LAG;
             break;
         case Port::VLAN:
-            bind_type = SAI_ACL_BIND_POINT_TYPE_VLAN;
+            sai_acl_bind_type = SAI_ACL_BIND_POINT_TYPE_VLAN;
```

Comment:
I don't think the support for VLAN ACLs exists in aclorch currently. Is this something that is needed for DPB?

Reply:
This was already part of existing code snippet, I moved it to this function. I think we should leave it as is, though it wont be exercised as we do NOT support binding ACL tables to VLANs today. But this will anyways be an essential work in future.